ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -232,6 +232,12 @@
     <p>A pre-populated email form will appear and you should add details to the message body. Remember: the author may only know specifics about this puzzle and not others in the event.</p>
 }
 
+@if (Model.AllowFeedback) 
```

Comment:
If you use Model.Event.AllowFeedback here, then you do not need to add a new property.

Reply:
Good catch!